Cargo.toml

```diff
@@ -13,3 +13,6 @@ async-h1 = "1.1"
 http-types = "1.1"
 async-std = "1.5.0"
 codegen = { path = "codegen" }
+
+[dev-dependencies]
+lazy_static = "1.4.0"
```

Comment:
Whats this for?

Reply:
before, there was a cargo.toml in the example dir with some dependencies. Since Cargo.toml is gone, you put all example deps into dev-dependencies.

I didn't actually check if it's needed. I just assumed it was needed in the example because it was in your Cargo.toml.